conan/tools/cmake/utils.py

```diff
@@ -4,14 +4,22 @@ def is_multi_configuration(generator):
     return "Visual" in generator or "Xcode" in generator or "Multi-Config" in generator
 
 
-def get_file_name(conanfile, find_module_mode=False):
+def get_file_name(conanfile, forced_module_mode=None):
     """Get the name of the file for the find_package(XXX)"""
     # This is used by the CMakeToolchain to adjust the XXX_DIR variables and the CMakeDeps. Both
     # to know the file name that will have the XXX-config.cmake files.
-    if find_module_mode:
+    if forced_module_mode or get_find_mode(conanfile) == "module":
```

Comment:
Should `FIND_MODE_MODULE` constant from [here](https://github.com/conan-io/conan/blob/develop/conan/tools/cmake/cmakedeps/cmakedeps.py#L14) be used? Should other hardcoded strings in this file be fixed too?